### DISCUSSION TITLE
PR hourly limit remaining: 0

### DISCUSSION THREAD

**Discussion by janpio** [renovatebot/renovate#8716] [Request Help] [answered]
For some reason a new repo where we are using the Renovate CLI gets this in the Dependency Dashboard issues:

> Rate Limited
> These updates are currently rate limited. Click on a checkbox below to force their creation now.

In the logs I can find this:

```
...
DEBUG: Fetching changelog: renovatebot/renovate (24.49.9 -> 24.52.0) (repository=prisma/prisma-version-monitor)
DEBUG: config.repoIsOnboarded=true (repository=prisma/prisma-version-monitor)
DEBUG: processRepo() (repository=prisma/prisma-version-monitor)
DEBUG: Processing 3 branches: renovate/integration-renovate-24.x, renovate/latest-prisma-monorepo, renovate/patch-dev-renovate-24.x (repository=prisma/prisma-version-monitor)
DEBUG: Calculating hourly PRs remaining (repository=prisma/prisma-version-monitor)
DEBUG: Retrieving PR list (repository=prisma/prisma-version-monitor)
DEBUG: Retrieved 3 Pull Requests (repository=prisma/prisma-version-monitor)
DEBUG: currentHourStart=2021-02-16T15:00:00.000+00:00 (repository=prisma/prisma-version-monitor)
DEBUG: PR hourly limit remaining: 0 (repository=prisma/prisma-version-monitor)
DEBUG: Calculating prConcurrentLimit (20) (repository=prisma/prisma-version-monitor)
...
```

How does it come to the conclusion of `PR hourly limit remaining: 0`? 

Our configuration does not use `prHourlyLimit` at all, so I would expect it to create as many PRs as possible.

From reading the code, I think it should return 99 here and use that to do the calculation: https://github.com/renovatebot/renovate/blob/44004bf2d6904e510e374d0437d56689014aa1f6/lib/workers/repository/process/limits.ts#L10-L41

**Comment by rarkins** [comment, marked as answer]
If you extend `config:base` then it is configured to create 2 PRs per hour maximum, excluding the onboarding PR. And if so then that would also have been described in the onboarding PR description when it was first created.

**Reply by janpio** [reply]
Interesting. Seems we did not get any onboarding PR for that repo even - as we started with a `renovate.json` file maybe?

I opened a PR to surface the actual value: https://github.com/renovatebot/renovate/pull/8720
